packages/core/admin/admin/src/components/AuthenticatedApp/utils/api.js

```diff
@@ -11,8 +10,12 @@ const fetchStrapiLatestRelease = async (toggleNotification) => {
   try {
     const {
       data: { tag_name },
-    } = await axios.get('https://api.github.com/repos/strapi/strapi/releases/latest');
-
+    } = await get('https://api.github.com/repos/strapi/strapi/releases/latest', {
```

Comment:
does this actually work? I thought this would be appended to the `baseUrl`?

Reply:
yes they work and it doesn't append the baseUrl